main/main.cpp

```diff
@@ -678,7 +678,8 @@ void Main::test_cleanup() {
 
 int Main::test_entrypoint(int argc, char *argv[], bool &tests_need_run) {
 #ifdef TESTS_ENABLED
-	for (int x = 0; x < argc; x++) {
+	int x;
```

Comment:
This just creates noise and polutes the scope, it does nothing productive

Reply:
i read somewhere that initialization before the loop slightly improves performance but i missed the part of if the variable is used for multiple loops so i will fix that